### ISSUE TITLE
tighten player up for touchscreen

### ISSUE THREAD

**Issue by increpare** [increpare/PuzzleScript#446]
``` I’m impressed at how well these games work on an iPhone. Any chance you could fix the view window so it doesn’t scroll around weird when swiping or pressing undo a bunch of times?```

https://twitter.com/islandguybyu/status/1005763407969017857

**Comment by lazerwalker** [comment]
@increpare Trying to play a bunch of puzzlescript games this week on my phone was bugging me, so I started investigating this even before I looked and saw this issue :P

A lot of the usual solutions (binding a `touchmove` event handler that `preventsDefault()`, setting CSS `overflow: hidden`, etc) weren't working for me, but what did reliably work was adding the following CSS to an element wrapping the entire page (either directly on `body`, or on a new `div` that wrapped all existing DOM nodes on the page):

```css
height: 100%;
overflow: hidden;
position: fixed;
width: 100%;
```

In the iPhone simulator (iOS 11.4), this consistently removes the vertical page scroll bounce when you swipe up or down.

Because of the `position: fixed`, you presumably don't want to add this in every case, only if on mobile (I'd assume using the mobile JS checks you already have).

Happy to dive into the code itself and turn this into a proper PR, but the Google Groups thread linked in the README suggested you'd probably rewrite yourself rather than just merge, so I figured it's probably easier to let you deal with getting this working? (Also happy to just jam on this in-person later this week :P)
